packages/hoppscotch-common/src/components/collections/graphql/Folder.vue

```diff
@@ -128,8 +128,10 @@
                   :shortcut="['P']"
                   @click="
                     () => {
+                      debugger
                       emit('edit-properties', {
                         collectionIndex: collectionIndex,
+                        // AM-REMINDER: check this line, because collection is not defined anywhere
```

Comment:
Is there a known issue here?

Reply:
not related to this PR. a type error found in the wild.